src/main/scala/firrtl/Emitter.scala

```diff
@@ -781,6 +790,14 @@ class VerilogEmitter extends SeqTransform with Emitter {
       lines += Seq("`endif // SYNTHESIS")
     }
 
+    def addFormal(clk: Expression, en: Expression, stmt: Seq[Any], info: Info) = {
```

Comment:
can we move this an `formalStatement` to the SystemVerilog emitter?

Reply:
Not unless we make a separate SystemVerilogRenderer too, which seems like over-kill. Plus, by leaving it in, if/when we want to have the Verilog emitter emit muxed `printf`s for asserts it will be easy to implement by simply updating the addFormalStatement method.

Reply:
OK. Then this is good enough for now.